Assets/MixedRealityToolkit/Utilities/AssetScriptReferenceRetargeter.cs

```
@@ -343,10 +364,164 @@ private static bool IsYamlFile(string filePath)
+                }
+
+                subfolderName = sourceToOutputPair.Value;
+
+                string pluginPath = Path.Combine(outputPath, subfolderName);
+                if (Directory.Exists(pluginPath))
+                {
+                    Directory.Delete(pluginPath);
+                }
+                Directory.CreateDirectory(pluginPath);
+
+                FileInfo[] dlls = directory.GetFiles("Microsoft.MixedReality.Toolkit*.dll", SearchOption.AllDirectories);
+                foreach (FileInfo dll in dlls)
+                {
+                    File.Copy(dll.FullName, Path.Combine(pluginPath, dll.Name), true);
+                }
+            }
+        }
+
+        private static void UpdateMetaFiles()
+        {
+            int lengthOfPrefix = Application.dataPath.IndexOf("Assets");
+            Dictionary<string, string> dllGuids = new Dictionary<string, string>();
+            string[] asmdefFiles = Directory.GetFiles(Application.dataPath, "*.asmdef", SearchOption.AllDirectories);
+
+            foreach (string asmdefFile in asmdefFiles)
+            {
+                string asmdefText = AssetDatabase.LoadAssetAtPath<AssemblyDefinitionAsset>(asmdefFile.Substring(lengthOfPrefix)).text;
+                string dllName = JsonUtility.FromJson<AssemblyDefinitionStub>(asmdefText).name;
+                string guid = File.ReadAllLines($"{asmdefFile}.meta")[1].Substring(6);
+                guid = CycleGuidForward(guid);
+                dllGuids.Add($"{dllName}.dll", guid);
+            }
+
+            //Load the sample meta files
+            DirectoryInfo assetDirectory = new DirectoryInfo(Application.dataPath);
+            FileInfo editorMetaFile = assetDirectory.GetFiles("*sampleEditorDllMeta.txt", SearchOption.AllDirectories).FirstOrDefault();
+            if (editorMetaFile == null)
+            {
+                Debug.LogError("Could not find sample editor dll.meta file");
```

> Debug.LogError("Could not find sample editor dll.meta file"); [](start = 15, length = 62)

We should throw an error here